### DISCUSSION TITLE
simulating long eksperiment

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3072] [Q&A]
"x_100", "y_100", "x_0", "y_0" that means ion concentration or what, sir? @brosaplanella 
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/a1becda1-98b9-4687-9ec4-6827e239acb9)

**Comment by brosaplanella** [comment]
These are the stoichiometries at 0% and 100% SOC for each electrode.
